src/app/app.component.css

```diff
@@ -0,0 +1,19 @@
+header {
```

Comment:
Define CSS classes in HTML and target them here instead of HTML tags.
It decouples the structure from the style